manager.py

```diff
@@ -55,17 +56,18 @@ def farm_manager(verbose=False, clean_reports=False):
                 perf = "Low Profile "
             if verbose:
                 logger.info(
-                    "%sFarm village %s attacked %d times - Total loot: %s - Total units lost: %s (%s)",
-                    perf, farm, len(num_attack), str(loot), str(total_loss_count), str(percentage_lost)
+                    "%sFarm village %s attacked %d times - Total loot: %s - Total units lost: %d (%.2f)",
+                    perf, farm, len(num_attack), str(loot), total_loss_count, percentage_lost
                 )
             if len(num_attack):
                 total = 0
                 for k in loot:
                     total += loot[k]
                 if len(num_attack) > 3:
-                    if total / len(num_attack) < 100 and (
+                    if (total / len(num_attack)
```

Comment:
How it was before is better readable
```suggestion
if total / len(num_attack) < 100 and (
    "low_profile" not in data or not data["low_profile"]
 ):
```